admin/src/components/ConfirmRegisterMailFormular.vue

```diff
@@ -0,0 +1,63 @@
+<template>
+  <div class="component-confirm-register-mail">
+    <div class="shadow p-3 mb-5 bg-white rounded">
+      <div class="h5">
+        Die letzte Email wurde am
+        <b>{{ dateLastSend }} Uhr</b>
+        an das Mitglied ({{ email }}) gesendet.
+      </div>
+
+      <!-- Using components -->
+      <b-input-group prepend="Email bestätigen, wiederholt senden an:" class="mt-3">
+        <b-form-input readonly :value="email"></b-form-input>
+        <b-input-group-append>
+          <b-button variant="outline-success" class="test-button" @click="sendRegisterMail">
+            Registrierungs-Email bestätigen, jetzt senden
+          </b-button>
+        </b-input-group-append>
+      </b-input-group>
+    </div>
+  </div>
+</template>
+<script>
+import { sendActivationEmail } from '../graphql/sendActivationEmail'
+
+export default {
+  name: 'ConfirmRegisterMail',
+  props: {
+    email: {
+      type: String,
+    },
+    dateLastSend: {
+      type: String,
+    },
+  },
+  methods: {
+    sendRegisterMail() {
+      this.$apollo
+        .mutate({
+          mutation: sendActivationEmail,
+          variables: {
+            email: this.email,
+          },
+        })
+        .then(() => {
+          // eslint-disable-next-line no-console
+          console.log(
+            'Erfolgreich senden der Confirmation Link an die E-Mail des Users!',
+            this.email,
+          )
+        })
+        .catch((error) => {
+          // eslint-disable-next-line no-console
+          console.log('Fehler beim senden des confirmation link an den Benutzer', error)
```

Comment:
toast this error

Reply:
Replaced in then with this.$toasted.global.success, and in catch this.$toasted.global.error